src/features/pages/404/content/not-found.container.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { button_link, info_items, not_available_image } from './data'
+import { button, container, info } from './not-found.module.scss'
+import Typography from 'features/components/atoms/typography'
+import Button from 'features/components/atoms/button'
+import Link from 'features/components/atoms/link'
+import Image from 'features/components/atoms/image'
+import Container from 'features/components/atoms/container'
+import { Localize } from 'components/localization'
+
+const NotFoundContainer = () => {
+    return (
+        <Container.Fixed className={container}>
+            <Image
+                src={not_available_image.src}
+                alt={not_available_image.alt}
+                width="96"
+                height="96"
+            />
+
+            <Container.Fixed className={info}>
```

Comment:
please use `FlexBox` component. you got all the styles you added with props there.